gui-libs/libpanel/libpanel-1.0.1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+VALA_MIN_API_VERSION="0.52"
+inherit gnome.org meson python-any-r1 vala virtualx
+
+DESCRIPTION="A dock/panel library for GTK 4"
+HOMEPAGE="https://gnome.pages.gitlab.gnome.org/libpanel/ https://gitlab.gnome.org/GNOME/libpanel"
+
+LICENSE="LGPL-2.1+"
+SLOT="1"
+IUSE="+introspection test +vala"
+REQUIRED_USE="vala? ( introspection )"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+RDEPEND="
+	>=dev-libs/glib-2.66:2
+	>=gui-libs/gtk-4.5.0:4[introspection?]
+	dev-libs/fribidi
+	>=dev-util/gi-docgen-2022.1
+	introspection? ( >=dev-libs/gobject-introspection-1.54:= )
+"
+DEPEND="${RDEPEND}
+	test? ( dev-libs/appstream-glib )"
+BDEPEND="
+	${PYTHON_DEPS}
+	vala? ( $(vala_depend) )
+	>=dev-util/meson-0.59.0
+	dev-util/glib-utils
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	default
+	use vala && vala_setup
+}
+
+src_configure() {
+	local emesonargs=(
+		# Never use gi-docgen subproject
+		--wrap-mode nofallback
+
+		$(meson_feature introspection)
+		$(meson_use vala vapi)
+	)
+	meson_src_configure
+}
+
+src_test() {
+	virtx meson_src_test --timeout-multiplier 2
+}
+
+src_install() {
+	meson_src_install
+
```

Comment:
this bulid installs the gtk-docs into `/usr/share/gtk-doc`, which causes QA issue because they belong to `/usr/share/gtk-doc`
Thankfully you can manipulate the image in "${ED}" before it gets merged into the actual system. 
So add this:
```
if use gtk-doc; then
		mkdir "${ED}"/usr/share/gtk-doc || die
		mv "${ED}"/usr/share/doc/panel-1.0 "${ED}"/usr/share/gtk-doc/ || die
	fi
```